python/tests/requirements.txt

```diff
@@ -9,4 +9,3 @@ pytest-cov==2.12.1
 pytest-benchmark==3.4.1
 pytest-flake8==1.0.7
 pytest-black==0.3.12
-gqlalchemy==1.3.2
```

Comment:
I think we need this requirement for tests to run

Reply:
For e2e tests: 
- name: Run End-to-end tests
        if: matrix.architecture != 'arm64'
        env:
          PYTHONPATH: "$PWD/e2e"
        run: |
          cd e2e
          python -m pytest . -k 'not cugraph'

Reply:
Ah, you are right, I didn't see that we are installing `requirements.txt`:

```
- name: Install Python test dependencies
        run: |
          python -m pip install --upgrade pip
          pip install -r ./python/tests/requirements.txt
```